installer/install.iss

```diff
@@ -2584,9 +2588,9 @@ begin
             if DirExists(AppDir) then begin
                 if not FileExists(ExpandConstant('{tmp}\blocked-file-util.exe')) then
                     ExtractTemporaryFile('blocked-file-util.exe');
-                Cmd:='"'+ExpandConstant('{tmp}\blocked-file-util.exe')+'" blocking-pids "'+AppDir+'"';
-                if not ExecWithCapture(Cmd,StdOut,StdErr,Res) or (Res<>0) then begin
-                    Msg:='Skipping installation because '+AppDir+' is still in use:'+#13+#10+StdErr;
+                Cmd:='"'+ExpandConstant('{tmp}\blocked-file-util.exe')+'"';
+                if not ExecAndCaptureOutput(Cmd, 'blocking-pids "'+AppDir+'"', '', SW_SHOWNORMAL, ewWaitUntilTerminated, Res, Output) or (Res<>0) then begin
```

Comment:
It might make sense to use AppDir as the working dir here

Reply:
Looking at https://github.com/git-for-windows/build-extra/blob/30a1958fea52277914b4d4439f1442d332ecd9f1/git-extra/blocked-file-util.c#L79-L110 ,
`blocked-file-util.exe` doesn't care about working dir.